contracts/Workers.sol

```diff
@@ -26,33 +26,42 @@ import "./EIP20Interface.sol";
 import "./OpsManaged.sol";
 import "./WorkersInterface.sol";
 
-/// A set of authorised workers
+/**
+ *  @title Workers contract which implement WorkersInterface, OpsManaged.
+ *
+ *  @notice A set of authorised workers.
+ */
 contract Workers is WorkersInterface, OpsManaged {
     using SafeMath for uint256;
+
+    /** Constants */
     
     EIP20Interface private eip20token;
-    /*
-     *  Storage
-     */
-    /// workers are active up unto the deactivation height
+    
+    /**  Storage */
+    /** Workers are active up unto the deactivation height. */
```

Comment:
Realizing this is the language from before, it appears incorrect 🤔 isWorker returns true if the worker's deactivation height matches the current block. It's not until after the deactivation height that a worker is no longer active.

Suggest (feel free to come up with something better): "Workers are not active once the deactivation height is passed."

Reply:
Done!